pkg/controllers/jobexecution_controller.go

```diff
@@ -75,7 +75,7 @@ func (r *JobExecutionReconciler) Reconcile(ctx context.Context, req ctrl.Request
 	}
 
 	if job == nil {
-		if je.Status.Phase == v1alpha1.JobExecutionCompletedPhase {
+		if je.Status.Phase == v1alpha1.JobExecutionCompletedPhase || je.Status.Phase == v1alpha1.JobExecutionFailedPhase {
```

Comment:
Would it be best to separate these conditions? i.e. "JobExecution has Failed: {some job context like job name, and payload?}".

Reply:
I'm not sure I understand how you're proposing to separate this, can you give me an example?

Reply:
Ah, got you. See 20a783e.

Reply:
Yes, this was what I meant 👍🏽